.buildkite/pipeline.yaml

```diff
@@ -11,6 +11,7 @@
   artifact_paths:
     - "dist/*.AppImage"
     - "cypress/screenshots/**/*.png"
+    - "cypress/workspace/test-tmp/*/node-*/*.log"
```

Comment:
Maybe for a follow-up issue: we could also upload a snapshot of a `tree` output of the monorepo, so that we can diagnose link problems better. Or even tar all the state and upload that on spec failure?

Reply:
Good idea. I’ll create a follow-up issue.